src/modules/Dashboard/features/ParticipantDetails/ParticipantActivities.tsx

```diff
@@ -0,0 +1,3 @@
+export const ParticipantActivities = () => <div>Activities tab</div>;
```

Comment:
My suggestion is to move these components to their own directories (features/ParticipantActivities, features/ParticipantConnections, features/ParticipantSchedule).